apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -12,6 +13,34 @@ export const COLUMNS = {
   NOT_ANSWERED: 3,
 };
 
+const answerOptionsFormatter = (percentAnsweredOptionOne, {rowData}) => {
+  return (
+      <MultipleChoiceAnswerCell
+        id={rowData.id}
+        percentValue={rowData.percentAnsweredOptionOne}
+        isCorrectAnswer={rowData.percentAnsweredOptionTwo}
```

Comment:
`isCorrectAnswer` is meant to be a boolean. Instead of setting it for a particular column, you'll want to have information about if an answer is the correct answer in your prop shape that you define, and then pass in the data via storybook.
Something like `optionOneIsCorrect: true` and `optoinTwoIsCorrect: false`

Then in your cellFormatter for your answer cells, something like: 
```
<MultipleChoiceAnswerCell
  id={rowData.id}
  percentValue={rowData.percentAnsweredOptionOne}
  isCorrectAnswer={rowData.optionOneIsCorrect}
/>
```

Since you will have to allow for a variable amount of answers, it might be best to go ahead and have your answers in an array, where they can each store this data, and then all use the same formatter.